src/Microsoft.VisualStudio.Editors/OptionPages/OptionPage.vb

```diff
@@ -0,0 +1,74 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.ComponentModel
+Imports System.Windows
+Imports Microsoft.VisualStudio.Shell
+
+Namespace Microsoft.VisualStudio.Editors.OptionPages
+    ' This code is based on corresponding functionality in dotnet\roslyn: https:'github.com/dotnet/roslyn/blob/master/src/VisualStudio/Core/Impl/Options/AbstractOptionPage.cs
+
+    <DesignerCategory("code")>
+    Public MustInherit Class OptionPage
```

Comment:
I can make this Friend...

---
In reply to: [117557708](https://github.com/dotnet/project-system/pull/2243#discussion_r117557708) [](ancestors = 117557708)